test/pg-cdc-resumption/add-relation-to-publication.td

```diff
@@ -0,0 +1,13 @@
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+$ postgres-execute connection=postgres://postgres:postgres@postgres
+ALTER PUBLICATION mz_source ADD TABLE t3;
```

Comment:
I think for a most-stressful test, t3 should contain data prior to adding it to the publication, if that scenario is supported.

In this case, the INSERT below can be replaced with a DELETE, in the pattern of the ```delete*.td``` tests.

Reply:
Done! Thank you for the suggestion!